settings.gradle

```diff
@@ -15,3 +15,5 @@
 
 include 'core', 'examples', 'clients', 'tools', 'streams', 'streams:examples', 'log4j-appender',
         'connect:api', 'connect:transforms', 'connect:runtime', 'connect:json', 'connect:file', 'jmh-benchmarks'
+include 'log4j-controller'
```

Comment:
Alright. I'll undo the module separation then but leave the dynamic class load, so people won't run into any classload problems when excluding the log4j jar.